src/Blockcore.Indexer.Cirrus/Models/QueryAddressContract.cs

```diff
@@ -0,0 +1,16 @@
+namespace Blockcore.Indexer.Cirrus.Models
+{
+   public class QueryAddressContract
```

Comment:
Would be nice if this model could be documented? What are the different properties, what are allowed values, what is possible values?

Reply:
I do agree however it is a bit of an early draft as I am not sure how the db should look like eventually, I can put some comment though good idea